HW06.Rmd

```diff
@@ -77,12 +98,39 @@ Now create a graphing function to create boxplots to visualize the results. Plot
 
 ```{r create a graphing function}
 #to work in the tidyverse, it will be easier to make tidy the dataframe first
+#Create a data set of values from H3K27_Mutated group only and reshape the data set from wide to long
+H3K27_Mutated <- GBM.genes.of.interest %>%
+  dplyr::select(gene_symbol, TPM...4:TPM...114) %>%
+  pivot_longer(cols = TPM...4:TPM...114) %>%
+  dplyr::select(gene_symbol, TPM = value) %>%
+  mutate(Group = "H3K27_Mutated")
+

```

Comment:
pivot_longer() is another useful function that I did not know before reviewing your homework!